code/modules/mining/equipment/marker_beacons.dm

```diff
@@ -124,8 +124,16 @@ GLOBAL_LIST_INIT(marker_beacon_colors, list(
 			M.add(1)
 			playsound(src, 'sound/items/deconstruct.ogg', 50, 1)
 			qdel(src)
-	else
-		return ..()
+			return
+	if(istype(I, /obj/item/light_eater))
+		var/ash_type = /obj/effect/decal/cleanable/ash
+		var/obj/effect/decal/cleanable/ash/A = new ash_type(loc)
+		A.desc += "\nLooks like this used to be \an [name] some time ago."
```

Comment:
Isn't there a proc for generating ash like this

Reply:
The proc takes items, this is a structure. Therefore, this